content/about/notes/1.1/index.md

```diff
@@ -56,6 +56,11 @@ to create global namespaces. Locality-aware routing is enabled by default in the
 [Securing Kubernetes Ingress with Cert-Manager](/docs/examples/advanced-gateways/ingress-certmgr/) example for more details on how
 to use Kubernetes Ingress resources with [gateways](/docs/concepts/traffic-management/#gateways).
 
+- **Behavioral changes**.
+    - Outbound traffic policy is set to `ALLOW_ANY` by default. As a result, traffic to unknown ports will be forwarded as-is. Traffic to known ports (e.g., port 80) will be matched with one of the services in the system and forwarded accordingly.
+    - Destination rule resolution order has been set. For a given sidecar, when routing to a particular service, destination rules for the target service in the same namespace as the sidecar will first be considered, followed by destination rules in the service’s namespace, and finally followed by destination rules in `configRootNamespace` if defined or any other namespace.
```

Comment:
destination rules for the target service in the same namespace as the sidecar will first be considered
--> destination rules for the target service in the same namespace as the source sidecar will first be considered?

followed by destination rules in the service’s namespace
-> followed by destination rules in the target service’s namespace?

Reply:
> and finally followed by destination rules in `configRootNamespace` if defined or any other namespace.

The `or any other namespace` part is confusing. You mean `or any other namespace if configRootNamespace is not set`.

Didn't we make `configRootNamespace` default to `istio-system` anyway, so it would need to be explicitly unset for the any namespace lookup to kick in?

Reply:
nope.. it was not set to istio-system

Reply:
https://github.com/istio/istio/pull/12234

What this reverted?